enclave_manager/avalon_enclave_manager/worker_kv_delegate.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import hashlib
+
+
+logger = logging.getLogger(__name__)
+
+
+class WorkerKVDelegate:
+    """
+    Delegate class for enclave managers to make worker
+    specific changes in the KV storage.
+    """
+
+    def __init__(self, kv_helper):
+        self._kv_helper = kv_helper
+
+    def cleanup_worker(self):
```

Comment:
You could probably rename this function to `cleanup_all_workers()` since it removes all workers. There can a different `cleanup_worker()` function which takes worker_id or list of worker_ids as argument.

Reply:
Makes sense. Shall I add a list(default None) to this function itself similar to cleanup_work_orders ? In the absence of a specific list, the function would default to cleaning up  all.